### PR TITLE
handle multiple bars

### PR DESCRIPTION
**Pull request by n0pan**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,26 +12,28 @@ Toggle the extension on and you are ready to go.
...
+| breakAnkles         | teleport around the map and break your coworker's ankles   |
+| desk                | teleport to your desk position                             |
+| getMap              | get the current map                                        |
+| getMaps             | returns all the maps for your environment                  |
+| getPlayers          | teleports your avatar in a selected space                  |
+| listMaps            | prints all the mpas                                        |
+| listPlayers         | teleports your avatar in a selected space                  |
+| position            | get the current position                                   |
+| setDesk             | save current position as desk position                     |
+| getMapsWithItemName | get maps containing object(s) with a certain name          |
```
Comment (michmich112) [inline README.md:26]: `getMapsWithItemName` is new right?

File: teleport.js [hunk 2]
```diff
@@ -134,6 +134,23 @@ function getPlayers() {
...
+function getMapsWithItemName(itemName) {
+  let mapsWithItemName = [];
+  wrapper((gameSpace) => {
+    const mapKeys = Object.keys(gameSpace.maps);
+    const maps = gameSpace.maps;
+    mapKeys.forEach(key => {
+      if (maps[key].objects.filter(o => (o._name || '').includes(itemName)).length > 0) {
+        mapsWithItemName.push(gameSpace.maps[key]);
+      }
+    })
```
Comment (michmich112) [inline teleport.js]: `mapsWithItemName =  mapKeys.filter(key => map[key].objects.filter(o => (o._name || '').include(itemName)).length > 0).map(key => gameSpace.maps[key])` should be sufficient

Comment (michmich112) [reply teleport.js]: and you can also empty initialize mapsWIthItemName

**Review by michmich112 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (n0pan) [reply README.md:26]: yes

**Review by michmich112 — APPROVED** [review]

**Pull request merged**